src/shared/util.js

```diff
@@ -709,6 +709,17 @@ class FeatureTest {
       typeof OffscreenCanvas !== "undefined"
     );
   }
+
+  static get isCSSColorCanvasTextSupported() {
+    return shadow(
+      this,
+      "isCSSColorCanvasTextSupported",
+      typeof CSS !== "undefined" &&
+        typeof CSS.supports === "function" &&
+        CSS.supports("color", "CanvasText") &&
+        CSS.supports("color", "Canvas")
+    );
+  }
```

Comment:
Given that this is only used in *one* spot in the viewer, and is also completely necessary in the worker-thread, you can probably just place this as a regular function in the `web/base_viewer.js`-file instead.

Also, looking at https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports#browser_compatibility and https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support I don't believe that you need to check for `CSS.supports` being available.